tests/test_octodns_provider_cloudflare.py

```diff
@@ -153,7 +153,7 @@ def test_populate(self):
 
             changes = self.expected.changes(zone, provider)
 
-            self.assertEquals(0, len(changes))
+            self.assertEquals(1, len(changes))
```

Comment:
The intention of this test appears to be that `0` changes happen. It should be possible to get that to be the case by adding the root NS record to the fixture and/or in-code records setup. This appears in a few other providers too.